README.md

```diff
@@ -203,9 +203,9 @@ To run test searches, you will need a working Bing API key. You can request one
   bing_v7:
     web_subscription_id: *****
 ```
-2. Start your local rails server:
+2. Start your local development environment:
```

Comment:
Running `assets:precompile` is needed only for production environments. I added a note at the end of this README but it might be better for it to be documented in the `search-infra` repo.